spec/models/billable_metric_spec.rb

```diff
@@ -7,20 +7,25 @@
 
   it_behaves_like "paper_trail traceable"
 
-  it { is_expected.to belong_to(:organization) }
+  describe "associations" do
+    it do
+      expect(subject).to belong_to(:organization)
+
```

Comment:
why this style change... I tend to prefer the shorthand version here with one test for each association.